src/nagini_translation/lib/constants.py

```diff
@@ -23,7 +23,9 @@
             'type',
             'list']
 
-BUILTIN_PREDICATES = ['list_pred', 'set_pred', 'dict_pred']
+THREADING = ['Thread']
+
+BUILTIN_PREDICATES = ['list_pred', 'set_pred', 'dict_pred', 'MayStart', 'ThreadPost']
```

Comment:
I don't really think so. Here, we basically categorize the things from contracts.py, so we definitely have to mention them here (because there's strictly more information here). On the other hand, we cannot just refer to the things here from contracts.py, since it should be possible to use the contracts library without having the translation part. If you have any ideas how to solve this, please let me know.